cockatrice/src/gameselector.h

```diff
@@ -23,6 +23,7 @@ private slots:
     void actCreate();
     void actJoin();
     void checkResponse(const Response &response);
+    void updateButtonChoices(const QModelIndex &);
```

Comment:
Nope; afaik github actually has no way to force review by external peers.
It typically boils down to th peer leaving a 👍  on the PR or commenting it.